src/aind_data_transfer_service/templates/job_status.html

```diff
@@ -26,7 +26,9 @@
     <nav>
         <a href="/">Submit Jobs</a> |
         <a href="/jobs">Job Status</a> |
-        <a title="Download job template as .xslx" href= "/api/job_upload_template" download>Job Submit Template</a>
+        <a title="Download job template as .xslx" href= "/api/job_upload_template" download>Job Submit Template</a> |
+        <a title="List of project names" href= "http://aind-metadata-service/project_names" >Project Names</a> |
```

Comment:
nit: open in new tab?

```suggestion
        <a title="List of project names" href= "http://aind-metadata-service/project_names" target="_blank" >Project Names</a> |
```